sklearn/model_selection/_search.py

```diff
@@ -444,9 +444,9 @@ def score(self, X, y=None):
             Input data, where n_samples is the number of samples and
             n_features is the number of features.
 
-        y : array-like, shape = [n_samples] or [n_samples, n_output], optional
-            Target relative to X for classification or regression;
-            None for unsupervised learning.
+        y : array-like or sparse matrix, shape = [n_samples] or
+            [n_samples, n_output], optional. Target relative to X for
```

Comment:
I believe that this breaks formatting of numpydoc: the first line must contain all the typing information, ie stop after "optional".